include/mbgl/util/image.hpp

```diff
@@ -26,6 +28,18 @@ class Image {
           height(h),
           data(std::move(data_)) {}
 
+    Image(Image&& o)
+        : width(o.width),
+          height(o.height),
+          data(std::move(o.data)) {}
+
+    Image& operator=(Image&& o) {
```

Comment:
There's actually some places where the default copy operator (now deleted) was in use:

https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/sprite/sprite_parser.cpp#L96